apps/judicial-system/backend/src/app/modules/file/file.service.ts

```diff
@@ -236,33 +239,41 @@ export class FileService {
       })
   }
 
+  private shouldGetConfirmedDocument = (file: CaseFile, theCase: Case) => {
+    // Only case files in indictment cases can be confirmed
+    if (!isIndictmentCase(theCase.type)) {
+      return false
+    }
+
+    // Only indictments that have been submitted to court can be confirmed
+    if (
+      file.category === CaseFileCategory.INDICTMENT &&
+      !hasIndictmentCaseBeenSubmittedToCourt(theCase.state)
+    ) {
+      return false
+    }
+
+    // Rulings and court records are only confirmed when a case is completed
+    if (
+      (file.category === CaseFileCategory.RULING ||
+        file.category === CaseFileCategory.COURT_RECORD) &&
+      !isCompletedCase(theCase.state)
+    ) {
+      return false
+    }
+
+    return true
```

Comment:
This will result in the confirmation of all other case file categories in indictments. I suggest modifying the two case file category conditionals above so that you can explicitely return `true` there and the return `false` here at the end of the functionl

Reply:
Ah! Good catch thanks 🙏